TMDBApp/TMDBApp/General.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Geral.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 19/12/2016.
+//  Copyright © 2016 Richard Leh. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SVProgressHUD
+
+typealias CompletionResultError = (_ result: Any?, _ error: Error?) -> Void
+typealias DictionaryData = Dictionary<String, Any>
+
+func updatesOnMain(_ updatesToMake: @escaping () -> Void) {
```

Comment:
I didn't know about it. The compiler optimize the build. It get all the source files in a module and it increase the performance of external functions.